plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -48,6 +48,7 @@
 import javax.naming.ConfigurationException;
 import javax.xml.datatype.XMLGregorianCalendar;
 
+import com.cloud.utils.script.Script;
```

Comment:
is this change on purpose?

Reply:
I had initially done some refactor to the logic of fetching id_rsa file and then reverted it - this is probably a consequence of that action